website/integrations/services/omni/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Integrate with Omni
+sidebar_label: Omni
+support_level: community
+---
+
+## What is Omni
+
+> Omni manages Kubernetes on bare metal, virtual machines, or in a cloud.
+>
+> -- https://github.com/siderolabs/omni
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `omni.company` is the FQDN of the Omni installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::note
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
+:::
+
+## authentik configuration
+
+To support the integration of _Omni_ with authentik, you need to create a property mapping and application/provider pair in authentik.
+
+### Create a Property Mapping, Application and Provider in authentik
```

Comment:
You could divide the two things into two diff H3 (###) sections, one for creating the property mapping, and the other for creating the app/provider pair, but up to you, this works too!